File: studies/tiny/ggplot2/slope_chart.R
library(tibble)
library(ggplot2)

df <- tribble(
  ~Contestant, ~Fruit, ~`Number Eaten`,
  "Alex", "Apples", 2,
  "Alex", "Oranges", 1,
  "Alex", "Bananas", 3,
  "Jordan", "Apples", 1,
  "Jordan", "Oranges", 3,
  "Jordan", "Bananas", 2
)

ggplot(df, aes(x = factor(Contestant), y = `Number Eaten`, color = factor(Fruit))) +
  geom_line(aes(group = factor(Fruit)))

File: galleries/tiny/holoviews/slope_chart.py
import holoviews as hv
import pandas as pd

df = pd.DataFrame(
    {
        "Fruit": ["Apples", "Oranges", "Bananas", "Apples", "Oranges", "Bananas"],
        "Contestant": ["Alex", "Alex", "Alex", "Jordan", "Jordan", "Jordan"],
        "Number Eaten": [2, 1, 3, 1, 3, 2],
    }
)

p = hv.Dataset(df).to(hv.Curve, "Contestant", "Number Eaten").overlay("Fruit")
